src/usr/local/www/diag_confbak.php

```diff
@@ -206,9 +206,18 @@
 print($form);
 
 if (is_array($confvers)) {
-	print_info_box(gettext('To view the differences between an older configuration and a newer configuration, ' .
-						   'select the older configuration using the left column of radio options and select the newer configuration in the right column, ' .
-						   'then press the "Diff" button.'));
+?>
+<div>
+	<div id="infoblock">
```

Comment:
Just pushed that. You may now use
````html5
<div id="infoblock_open" . . 
````
To have the infoblock open on initial page load